nanoFramework.Hardware.Esp32/Touch/PinValueChangedEventHandler.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace nanoFramework.Hardware.Esp32.Touch
+{
+    /// <summary>
+    /// Delegate function for the event handler.
+    /// </summary>
+    /// <param name="sender">The sender <see cref="TouchPad"/>.</param>
+    /// <param name="e">The touch arguments.</param>
+    public delegate void PinValueChangedEventHandler(
```

Comment:
I'm thinking this should be named differently... not only to make it clear that's a TOUCH event but also to disambiguate from the _real_ `PinValueChangedEventHandler` that lives in Sys.Dev.Gpio.

Reply:
renamed, should indeed be better